README.md

```diff
@@ -15,12 +15,12 @@ applications. Some of the features demonstrated in this example are:
 - Example CI configuration (using Github Actions)
 - Custom [west extension][west_ext]
 
-This repository is versioned together with the [Zephyr main tree][zephyr]. This
-means that every time that Zephyr is tagged, this repository is tagged as well
+This repository is versioned together with the [nRF Connect SDK main tree][sdk-nrf]. This
+means that every time that nRF Connect SDK is tagged, this repository is tagged as well
 with the same version number, and the [manifest](west.yml) entry for `zephyr`
-will point to the corresponding Zephyr tag. For example, the `example-application`
-v2.6.0 will point to Zephyr v2.6.0. Note that the `main` branch always
-points to the development branch of Zephyr, also `main`.
+will point to the corresponding nRF Connect SDK tag. For example, the `ncs-example-application`
+v2.3.0 will point to nRF Connect SDK v2.3.0. Note that the `main` branch always
```

Comment:
Why not 2.5.0?

Reply:
note this is just an example, doesn't need to be kept up to date